ReactWindows/ReactNative.Net46.Tests/ReactNative.Net46.Tests.csproj

```diff
@@ -153,7 +153,14 @@
       <Name>ReactNative.Net46</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\test.js">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </EmbeddedResource>
+  </ItemGroup>
   <Import Project="..\ReactNative.Shared.Tests\ReactNative.Shared.Tests.projitems" Label="Shared" />
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>xcopy /y "$(SolutionDir)..\ChakraCore\Build\VcBuild\bin\$(Platform)_$(Configuration)\ChakraCore.dll" "$(TargetDir)"</PostBuildEvent>
+  </PropertyGroup>
```

Comment:
Instead of using xcopy, can you use the MSBuild target...
```
<Target Name="AfterBuild">
  <ItemGroup>
    <ChakraDependencies Include="$(ChakraPath)\ChakraCore.*" />
  </ItemGroup>
  <Copy SourceFiles="@(ChakraDependencies)" DestinationFolder="$(OutputPath) " />
</Target>
```